app/assets/javascripts/comfortable_mexican_sofa/admin/modules/EmbeddedGoogleAnalytics.js

```diff
@@ -0,0 +1,108 @@
+define(['jquery', 'DoughBaseComponent', 'eventsWithPromises', 'async!https://apis.google.com/js/platform.js!onload'], function($, DoughBaseComponent, eventsWithPromises) {
+  'use strict';
+
+  var defaultConfig = { };
+
+  function EmbeddedGoogleAnalytics($el, config) {
+    EmbeddedGoogleAnalytics.baseConstructor.call(this, $el, config, defaultConfig);
+  }
+
+  /**
+   * Inherit from base module, for shared methods and interface
+   */
+  DoughBaseComponent.extend(EmbeddedGoogleAnalytics);
+
+  EmbeddedGoogleAnalytics.componentName = 'EmbeddedGoogleAnalytics';
+
+  EmbeddedGoogleAnalytics.prototype.init = function(initialised) {
+    this.clientid = '46744299348-7t8tn92mh4pv01vk89llfv0u3q7kup2j.apps.googleusercontent.com';
```

Comment:
Sorry I meant move this into `defaultConfig` above – meaning it can be overridden easily